manifests/entandocompositeapps.entando.org.crd.yaml

```diff
@@ -36,25 +36,23 @@ spec:
                 dbmsOverride:
                   description: >
                     The database management system (DBMS) to use for persistence for every component deployed in
-                    this EntandoCompositeApp. Can be any of mysql, oracle, postgresql or embedded. Each component
-                    deployed in the EntandoCompositeApp will be evaluated and if it supports the standard 'spec.dbms'
-                    property, the value of this property will be given to the component's 'spec.dbms'. Please consult
-                    the documentation for each component's CRD to determine the exact semantics of this value.
+                    this EntandoCompositeApp. Only 1 replica of the container is supported if embedded is selected. If
+                    Oracle is selected it must be created manually, the operator does not deploy Oracle instances.
```

Comment:
"If Oracle is selected, the database instance must be created manually since the Operator does not deploy Oracle instances."